Necrosis/Necrosis.lua

```diff
@@ -1626,7 +1506,7 @@ function Necrosis:UpdateMana()
 	-- If shadow guardian cooldown we gray || Si cooldown de gardien de l'ombre on grise
 	if _G["NecrosisBuffMenu8"] and self.Spell[43].ID then
 		local start, duration = GetSpellCooldown(self.Spell[43].ID, "spell")
-		if self.Spells[43].Mana > mana and start > 0 and duration > 0 then
+		if self.Spell[43].Mana > mana and start > 0 and duration > 0 then
```

Comment:
All these mana calculations will not work I believe. The mana values will not always reflect actual cost. All spells have a mana cost of 50 in the object and we retrieve the basic spell info. See Healthstone for instance:
```
[52] = {Name = GetSpellInfo(5699),	Mana = 50,				Length = 0,		Type = 0}, -- Create Healthstone
```
The ID referenced here is for https://classic.wowhead.com/spell=5699/create-healthstone
What if character is higher level and needs to cast https://classic.wowhead.com/spell=11729/create-healthstone-greater?